csplat.y

```diff
@@ -21,18 +21,31 @@ void yyerror (char const *err) { fprintf(stderr, "yyerror: %s\n", err); exit(-1)
 
 %%
 
-program: stmts {}
+program: stmts { printf("program -> stmts\n");}
+| program stmt { printf("program -> program stmt\n");}
 
-stmts: stmt {}
+stmts: add_exp ASSIGN { printf("stmts -> add_exp ASSIGN\n");}
```

Comment:
How are these related to "statements"? Each of them are one statement aren't they?

Reply:
idk, it's already named stmts when i get it